build/charts/antrea/values.yaml

```diff
@@ -393,10 +393,6 @@ webhooks:
 # -- Global log verbosity switch for all Antrea components.
 logVerbosity: 0
 
-whereabouts:
-  # -- Install and configure Whereabouts, for use by the antrea-agent.
-  enable: false
-
```

Comment:
should we keep this for now (with an updated comment) and produce an error if the the value is set to True? Or do you think this is not necessary specifically for this value (given that it is tied to the SecondaryNetwork feature, which is still Alpha)?

Reply:
Do you mean we have some rules to keep values backward compatible? In reality, I doubt anyone is setting this.

Reply:
> Do you mean we have some rules to keep values backward compatible

Well we should, even if we don't have documentation for this. Same as for configuration keys in the Antrea config.
I think unknown values will just be silently ignored by Helm. I would say for a typical value, we would need a deprecation phase, with a warning message. For this specific value, given that it's unlikely that it was used by anyone and given that it is tied to an Alpha feature (which is also not documented?), I am fine with removing it right away or adding an error if it is used (that's not backward compatible either, but at least it's not silently ignored).

It's definitely a good thing that we are removing this whereabouts binary from the image. Most vulnerabilities reported by CVE scanners are because of whereabouts...

Reply:
Could you check?